packages/clay-card/src/CardWithUser.tsx

```diff
@@ -7,6 +7,7 @@ import {ClayDropDownWithItems} from '@clayui/drop-down';
 import {ClayCheckbox} from '@clayui/form';
 import ClayIcon from '@clayui/icon';
 import ClayLabel from '@clayui/label';
+import * as ClayLayout from '@clayui/layout';
```

Comment:
This one is taken as an example, I've seen similar cases throughout the commit, but we seem to be importing all and then just using one subcomponent. In this case `<ClayLayout.ContentCol` on lines 140 & 163

Reply:
Just updated to have a default export so that we don't have to do this